index.css

```diff
@@ -0,0 +1,661 @@
+  font-size: 16px;
+  color: #d3d3d3;
+  padding-top: 37px;
+  padding-bottom: 5px;
+}
+
+hr.line-m {
+  width: 100px;
+  border: 1px solid red;
+}
+
+.programs {
+  display: flex;
+  flex-direction: column;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 90%;
+}
+
+.programs > div {
+  display: flex;
+  justify-content: center;
+  width: 95%;
+  align-items: center;
+  background-color: #272a31;
+  margin-bottom: 17px;
+}
+
+.programs > div > i {
+  opacity: 0.7;
+  color: #fff;
+  font-size: 30px;
+}
+
+.programs > div > p {
+  text-align: justify;
+  padding-right: 10px;
+  font-family: "lato", sans-serif;
+  font-size: 16px;
+  color: #d3d3d3;
+}
+
+.programs > div > h3 {
+  font-family: "lato", sans-serif;
+  font-size: 18px;
+  color: #ec5242;
+}
+
+.bussiness-h1 {
+  margin-left: 20px;
+  margin-right: 26px;
+}
+
+.bussiness-h2 {
+  margin-left: 15px;
+  margin-right: 5px;
+}
+
+.bussiness-h3 {
+  margin-left: 15px;
+  margin-right: 37px;
+}
+
+.bussiness-h4 {
+  margin-left: 15px;
+  margin-right: 18px;
+}
+
+.bussiness-h5 {
+  margin-left: 10px;
+  margin-right: 45px;
+}
+
+.detail-prog {
+  margin-top: 10%;
+  margin-bottom: 10%;
+  justify-content: center;
+  align-items: center;
+}
+
+.join-desk {
+  display: none;
+}
+
+.join-mobile {
+  font-family: "lato", sans-serif;
+  font-size: 20px;
+  padding: 5%;
+  padding-left: 5%;
+  padding-right: 5%;
+  color: #d3d3d3;
+  background-color: #ec5242;
+  margin-left: 10%;
+  margin-right: 10%;
+  border-radius: 10px;
+  text-decoration: none;
+}
+
+/* feature speakers start  */
+
+.feature-speaker {
+  padding: 10px;
+}
+
+.partners-d {
+  display: none;
+}
+
+.center {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  text-align: center;
+  padding: 40px 0 40px 0;
+}
+
+.under-line {
+  align-items: center;
+  font-family: "lato", sans-serif;
+  font-size: 24px;
+  color: #272a31;
+  font-weight: bold;
+  text-align: center;
+  padding-bottom: 5px;
+}
+
+hr.line-speaker {
+  width: 80px;
+  border: 1px solid red;
+}
+
+.trainers {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  margin-top: 50px;
+}
+
+.profesionalist {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 40px;
+}
+
+.right {
+  margin-right: 10px;
+}
+
+.right > img {
+  width: 150px;
+  height: auto;
+}
+
+.left {
+  display: flex;
+  flex-direction: column;
+}
+
+.left > h2 {
+  font-family: "lato", sans-serif;
+  font-size: 24px;
+  color: #272a31;
+  text-align: left;
+  padding-bottom: 10px;
+  padding-left: 20px;
+}
+
+.title {
+  font-family: "lato", sans-serif;
+  font-size: 18px;
+  color: #ec5242;
+  text-align: left;
+  padding-bottom: 20px;
+  padding-left: 20px;
+}
+
+.resume {
+  font-family: "lato", sans-serif;
+  font-size: 14px;
+  color: #272a31;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  text-align: left;
+}
+
+/* feature spreaker end */
+.some-right {
+  display: none;
+}
```

Comment:
- [ ] Kindly display this element on the mobile view as it is design requirement:


https://user-images.githubusercontent.com/95412943/227714317-3bbfe4f1-46bd-4781-b1c5-2ad93fa6a052.mp4